config/initializers/devise.rb

```diff
@@ -12,7 +12,7 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = 'please-change-me-at-config-initializers-devise@example.com'
+  config.mailer_sender = 'please-change-me-at-config-initializers-devise@mylibrarynyc.org'
```

Comment:
Let's change that to 'passwordreset@mylibrarynyc.org'. Thanks Deepika!

Reply:
@thisisstephenbetts Done